db/migrate/20210426150053_add_la_funded_place_urn_to_schools.rb

```diff
@@ -0,0 +1,5 @@
+class AddLaFundedPlaceUrnToSchools < ActiveRecord::Migration[6.1]
+  def change
+    add_column :schools, :la_funded_place_urn, :string, index: true
```

Comment:
when we introduce Social Care Leavers i assume we want to use the same column? if so would this be called `other_urn` or to that extent. Or would we introduce another column such as `social_care_urn`?

Reply:
Using `provision_urn` now as discussed